book/index.qmd

```diff
@@ -72,38 +83,57 @@ bma
 as.BenchmarkAggr(bm)$friedman_test()
 ```
 
-In this (much more complex!) example we picked two tasks, three machine learning algorithms ('learners'), and compared their performance. We demonstrated using automated tuning to optimise the number of trees in a random forest (Chapter 4), as well as pipelines to impute missing data, collapse factor levels and to create stacked models (Chapter 5). We also showed basic features like loading learners (Chapter 2) and picking resampling strategies for benchmarking (Chapter 3). Finally we compared the performance of the models using the sample mean and then use statistical tests to see if one model performed significantly better than another (they did not!).
+In this (much more complex!) example we chose two tasks and two machine learning algorithms ("learners" in `r mlr3` terms).
+We used automated tuning to optimise the number of trees in the random forest learner (@sec-optimization) and a machine learning pipeline that imputes missing data, collapses factor levels, and creates stacked models (@sec-pipelines).
+We also showed basic features like loading learners (@sec-basics) and choosing resampling strategies for benchmarking (@sec-performance).
+Finally, we compared the performance of the models using the mean accuracy on the test set, and applied a statistical test to see if the learners performed statistically significantly different (they did not!).
```

Comment:
I don't think we need double definition of 'statistical' so suggest revert back to '...performed significantly different..."